mne/io/eeglab/tests/test_eeglab.py

```diff
@@ -37,6 +37,13 @@ def test_io_set():
         _test_raw_reader(read_raw_eeglab, input_fname=raw_fname_onefile,
                          montage=montage)
         raw = read_raw_eeglab(input_fname=raw_fname_onefile, montage=montage)
+        # test finding events in continuous data
+        event_id={'rt':1, 'square':2}
+        raw = read_raw_eeglab(input_fname=raw_fname_onefile, montage=montage,
+                              event_id=event_id)
```

Comment:
since it's a `mat` file after all. Maybe you can read the events in, modify them and write them to a temp file which you can use for testing if the warnings are raised.